development/compiling/compiling_with_mono.rst

```diff
@@ -11,16 +11,18 @@ Requirements
 - Mono 5.12.0 or greater
 - MSBuild
 - NuGet
-- pkg-config
+- **On Linux/macOS only:** pkg-config
 
 You may need to import necessary certificates for NuGet to perform HTTPS
 requests.
 
 The recommended method is to use **curl**'s CA (Certificate Autorities) certificate bundle.
 
-Run the following commands to download and import it.
-On Windows, you can run it from the Mono command line prompt (or the regular prompt if you added Mono's ``bin`` directory to your ``PATH`` environment variable)::
+Run the following commands to download and import it. On Windows, you can run it
+from the Mono command line prompt (or the regular prompt if you added Mono's
+``bin`` directory to your ``PATH`` environment variable)::
 
+    # If using PowerShell, replace `curl` with `curl.exe` below.
```

Comment:
For the curious, this is required because `curl` on PowerShell is an alias for the `Invoke-WebRequest` cmdlet. You need to use `curl.exe` to use the actual curl binary shipped with Windows 10.

Reply:
Wow.